components/cli/pkg/commands/setup/setup_create.go

```diff
@@ -68,10 +68,20 @@ func createEnvironment(cli cli.Cli) error {
 	switch value {
 	case celleryGcp:
 		{
-			isCompleteSetup, isBackSelected := util.IsCompleteSetupSelected()
+			isCompleteSetup, isBackSelected, err := isCompleteSetupSelected()
+			if err != nil {
+				return fmt.Errorf("failed to select gcp basic or complete, %v", err)
+			}
 			if isBackSelected {
 				return RunSetup(cli)
 			}
+			confirmed, _, err := util.GetYesOrNoFromUser("This will create a Cellery runtime on a gcp cluster. Do you want to continue", false)
+			if err != nil {
+				return fmt.Errorf("failed to get confirmation to create, %v", err)
+			}
+			if !confirmed {
+				os.Exit(0)
```

Comment:
we can try to handle this in a slightly better way, similar to the ExecuteTask() function in the cli interface - have a ExecuteUseInputRequiredTask() method which will get the input and exit in the CelleryCli implementation, and return the decision in the MockCli implementation. This would help when we are writing tests for this method in future. WDYT? However, this is not urgent.

Reply:
+1 for the suggestion. Will implement in a separate PR.